rpcpb/rpc.proto

```diff
@@ -1,14 +1,17 @@
-// Copied from https://github.com/ava-labs/avalanche-network-runner/blob/v1.1.0/rpcpb/rpc.proto
-// without gRPC gateway dependency
-
 syntax = "proto3";
 
-option go_package = "github.com/ava-labs/avalanche-network-runner-sdk;rpcpb";
+option go_package = "github.com/ava-labs/avalanche-network-runner;rpcpb";
```

Comment:
Can we keep `/avalanche-network-runner-sdk`?

Reply:
yeah just saw that